client/src/document/on-github.tsx

```diff
@@ -4,13 +4,13 @@ export function OnGitHubLink({ doc }: { doc: Doc }) {
   return (
     <div id="on-github" className="on-github">
       <SourceOnGitHubLink doc={doc}>
-        {doc.locale !== "de"
+        {doc.locale !== "fr"
```

Comment:
These changes need to be reverted.

Reply:
Ah, I was looking at my TS issue there and added it 🙈